### ISSUE TITLE
keycode(16) returns up-arrow symbol instead of 'shift'

### ISSUE THREAD

**Issue by greim** [timoxley/keycode#15]
A workaround is possible but code like this is no longer possible:

```js
var pressed = keycode(ev);
if ('shift' === pressed) {
  // do something
}
```

I can provide a PR later if needed. Re-ordering the `codes` object literal ought to fix it.

**Comment by timoxley** [comment]
Yep, I think that's the only solution that won't break compatibility.

**Comment by timoxley** [comment]
Solved in #16.
